YAMS-LIB/Program.cs

```diff
@@ -353,7 +353,17 @@ void AddAllSpritesFromDir(string dirPath)
                 new UndertaleSprite.TextureEntry() {Texture =  gmData.TexturePageItems[nameToPageItemDict["sItemPBombDrop_4"]] },
             }
         });
-            
+        //Torchlight TODO: add sprites
```

Comment:
I need to properly set this up with dotnet-format later, but for code comments, I currently have the style guide of having a space between first letter and the slashes